tests/components/zwave_js/test_light.py

```diff
@@ -35,6 +35,7 @@
 )
 
 HSM200_V1_ENTITY = "light.hsm200"
+ZDB5100_ENTITY = "light.zdb5100"
```

Comment:
Looks like the entities are named `light.matrix_office`, `light.matrix_office_2`, `light.matrix_office_3` etc.

```suggestion
ZDB5100_ENTITY = "light.matrix_office"
```